src/ResourceManager/KeyVault/Commands.KeyVault/Properties/Resources.resx

```diff
@@ -147,6 +147,9 @@ You can find the object ID using Azure Active Directory Module for Windows Power
   <data name="InvalidApplicationId" xml:space="preserve">
     <value>Invalid application Id.</value>
   </data>
+  <data name="InvalidObjectIdSyntax" xml:space="preserve">
+    <value>Cannot parse ObjectId into Guid.</value>
```

Comment:
Got it. Do we need similar for SID type as well to display in case SID parsing fails